### PR TITLE
Limit to one domain

### PR DESCRIPTION
**Pull request by alianw-ar**
Please provide details about this PR, what should it do, which issues does it fix,how should it be tested

## Screenshots (if appropriate):

### CODE REVIEW

File: app/Http/Requests/StoreDomainRequest.php [hunk 1]
```diff
@@ -33,9 +33,22 @@ public function rules()
                     $fail(__('Invalid Domain Name'));
                 }
             }, new UniqueDomainRule($this->account, $this->install),
+            function ($attribute, $value, $fail) {
+                $domains = $this->install->domains->all();
+                $count = 0;
+
+                foreach ($domains as $domain) {
+                    if ($domain->name !== 'Elvis.steercampaign.net') {
```
Comment (mabumusa1) [inline app/Http/Requests/StoreDomainRequest.php:41]: What is the goal of this script? Why do you have static values here?

**Review by mabumusa1 — CHANGES_REQUESTED** [review]
The code does not the requested changes

File: app/Http/Requests/StoreDomainRequest.php (hunk 1 above)
Comment (ali-chaudhry) [reply app/Http/Requests/StoreDomainRequest.php:41]: Or do I need to get it from config?

Comment (mabumusa1) [reply app/Http/Requests/StoreDomainRequest.php:41]: ```Elvis.steercampaign.net``` this is a value that comes from a seeder(fake data), whey it is in the code.

The solution is simple if($this->install->domains->count() > 2) return invalid, it should not take more than 15 minutes to solve!!!

File: resources/views/installs/domains/index.blade.php [hunk 2]
```diff
@@ -9,9 +9,11 @@
         <div class="card card-borded">
             <div class="card-header">
                 <h2 class="mb-0 card-title">{{ __('Domains & Redirect') }}</h2>
+                @if(count($install->domains) <= 1 )
```
Comment (mabumusa1) [inline resources/views/installs/domains/index.blade.php:12]: It should be less than 2 $install->domains->count() < 2

File: app/Http/Requests/StoreDomainRequest.php (hunk 1 above)
Comment (ali-chaudhry) [reply app/Http/Requests/StoreDomainRequest.php]: chenged

**Pull request closed without merging**